src/main/target/ANYFCF7/target.h

```diff
@@ -24,7 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
-#define USE_ESC_TELEMETRY
+#define USE_ESC_SENSOR
```

Comment:
One little thing: We've just moved all the `USE_DSHOT` into `src/main/target/common.h` (https://github.com/betaflight/betaflight/blob/master/src/main/target/common.h#L35). We should probably consider doing the same for `USE_ESC_SENSOR`. But this can be done in a future PR once this one has been merged.